.changeset/lemon-carpets-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": patch
+---
+
+Add MediaRenderer component
```

Comment:
don't need a changeset here -> this will all be one big changeset for `beta` release later

Reply:
removed